docs/src/prototypes/dropdowns/AsyncDropdownSearch.tsx

```diff
@@ -34,25 +34,55 @@ const createEntry = (): Entry => ({
 // Prototype Search Page View
 // ----------------------------------------
 class AsyncDropdownSearch extends React.Component<{}, SearchPageState> {
+  private searchTimer: number
+
   state = {
     loading: false,
     searchQuery: '',
     items: [],
     value: [],
   }
 
-  searchTimer: number
+  render() {
+    const { items, loading, searchQuery, value } = this.state
+
+    return (
+      <>
+        <Dropdown
+          fluid
+          items={items}
+          loading={loading}
+          loadingMessage={{
+            content: <Loader label="Loading..." labelPosition="end" size="larger" />,
+          }}
+          multiple
+          onSearchQueryChange={this.handleSearchQueryChange}
+          onSelectedChange={this.handleSelectedChange}
+          placeholder="Try to enter something..."
+          search
+          searchQuery={searchQuery}
+          toggleIndicator={false}
+          value={value}
+        />
+        <Divider />
+        <CodeSnippet mode="json" value={this.state} />
+      </>
+    )
+  }
 
-  handleSelectedChange = (e: React.SyntheticEvent, { searchQuery, value }: DropdownProps) => {
+  private handleSelectedChange = (
+    e: React.SyntheticEvent,
+    { searchQuery, value }: DropdownProps,
+  ) => {
     this.setState({ value: value as Entry[], searchQuery })
   }
 
-  handleSearchQueryChange = (e: React.SyntheticEvent, { searchQuery }: DropdownProps) => {
+  private handleSearchQueryChange = (e: React.SyntheticEvent, { searchQuery }: DropdownProps) => {
     this.setState({ searchQuery })
     this.fetchItems()
   }
 
-  fetchItems = () => {
+  private fetchItems = () => {
```

Comment:
What is purpose of these changes? Do we need them?
We don't have an agreement on visibility modifiers so let's revert these changes.

Reply:
> We don't have an agreement on visibility modifiers so let's revert these changes.

I don't get this part, it's pretty obvious to me that in Typescript class members that are not meant to be used outside the scope of the class are private.

> What is purpose of these changes? Do we need them?

I will revert the changes however because the prototype section does not have the JS version so you might argue the change does not bring any benefit here.

Reply:
opened this PR agains wrong branch, moved to: https://github.com/stardust-ui/react/pull/931/files#r258164858